tests/test_addon_detail.py

```diff
@@ -48,3 +48,18 @@ def test_contribute_button(selenium, base_url, variables):
     # verifies that utm params are passed from AMO to the external contribute site
     wait = WebDriverWait(selenium, 10)
     wait.until(expected.url_contains(variables['contribute_utm_param']))
+
+
+@pytest.mark.nondestructive
+def test_extension_permissions(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get('{}/addon/{}'.format(base_url, extension))
```

Comment:
We could use python 3 f strings here. https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python

Reply:
> 
> 
> @jrbenny35 we have quite a few tests that are still using this old string formatting method. I'd say we merge this PR as it is and open a new issue to address string formatting overall. What do you think?

dang, you sound like me now! Love it!!!!